README.md

```diff
@@ -24,6 +24,26 @@ tls_interop --client ${NSS_ROOT}/dist/Darwin15.6.0_cc_64_DBG.OBJ/bin/nss_bogo_sh
 To swap client and server, you need to run it twice.
 
 
+Cargo Test Instructions
+============================
+Some of the internal rust test cases run with "cargo test" assume readily built
+versions of nss, boringssl and openssl being available in the parent diretory.  
+The NSS shim is expected to be found at "../dist/Debug/bin/nss_bogo_shim".  
+The BoringSSL shim is expected to be found at "../boringssl/build/ssl/test/bssl_shim".  
+The OpenSSL shim is expected to be found at "../openssl/tests/ossl_shim/ossl_shim".  
```

Comment:
Maybe add that OpenSSL has to be configured with `enable-external-tests` for this to work.